src/hexagon.variables.css

```diff
@@ -389,7 +389,7 @@
   --dropdown-background-color: var(--theme-container-background-color);
   --dropdown-border-color: var(--theme-container-border-color);
   --dropdown-shadow-color: var(--theme-shadow-color);
-  --dropdown-spacing: 0;
+  --dropdown-spacing: 0.2em;
```

Comment:
The `0` spacing looked odd, I believe it was added because non-zero values didn't work (i.e. because of the issue this PR solves)